docs/azure_arc_jumpstart/azure_arc_app_svc/aks/aks_azure_apimgmt_arm_template/_index.md

```diff
@@ -0,0 +1,185 @@
+    --resource-group <Name of the Azure resource group> \
+    --name <The name of this deployment> \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_app_services_jumpstart/aks/arm_template/azuredeploy.json \
+    --parameters <The *azuredeploy.parameters.json* parameters file location>
+    ```
+
+    > **Note: Make sure that you are using the same Azure resource group name as the one you've just used in the `azuredeploy.parameters.json` file**
+
+    For example:
+
+    ```shell
+    az group create --name Arc-AppSvc-Demo --location "East US"
+    az deployment group create \
+    --resource-group Arc-AppSvc-Demo \
+    --name arcappsvc \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_app_services_jumpstart/aks/arm_template/azuredeploy.json \
+    --parameters azuredeploy.parameters.json
+    ```
+
+    > **Note: The deployment time for this scenario can take ~60-90 minutes**
+
+* Once Azure resources has been provisioned, you will be able to see it in Azure portal. At this point, the resource group should have **7 various Azure resources** deployed.
+
+    ![ARM template deployment completed](./01.png)
+
+    ![New Azure resource group with all resources](./02.png)
+
+## Windows Login & Post Deployment
+
+* Now that first phase of the automation is completed, it is time to RDP to the client VM using it's public IP.
+
+    ![Client VM public IP](./03.png)
+
+* At first login, as mentioned in the "Automation Flow" section above, the [_AppServicesLogonScript_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_app_services_jumpstart/aks/arm_template/artifacts/AppServicesLogonScript.ps1) PowerShell logon script will start it's run.
+
+* Let the script to run its course and **do not close** the PowerShell session, this will be done for you once completed. Once the script will finish it's run, the logon script PowerShell session will be closed, the Windows wallpaper will change and the Azure web application will be deployed on the cluster and be ready to use.
+
+    > **Note: As you will notices from the screenshots below, during the Azure Arc enabled app services environment, the _log-processor_ service pods will be restarted and will go trough multiple Kubernetes pod lifecycle stages. This is normal and can safely be ignored. To learn more about the various Azure Arc enabled app services Kubernetes components, visit the official [Azure Docs page](https://docs.microsoft.com/en-us/azure/app-service/overview-arc-integration#pods-created-by-the-app-service-extension).**
+
+    ![PowerShell logon script run](./04.png)
+
+    ![PowerShell logon script run](./05.png)
+
+    ![PowerShell logon script run](./06.png)
+
+    ![PowerShell logon script run](./07.png)
+
+    ![PowerShell logon script run](./08.png)
+
+    ![PowerShell logon script run](./09.png)
+
+    ![PowerShell logon script run](./10.png)
+
+    ![PowerShell logon script run](./11.png)
+
```

Comment:
These screenshots are not good. They are kinda "cut" and resolution is weird.

Reply:
changed